### DISCUSSION TITLE
rename my username

### DISCUSSION THREAD

**Discussion by YangZxi** [community/community#40125] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello. I'm trying to rename my username to my previous name and I get this error:
"Username change was not successful. The repository name name/repo has been retired and cannot be reused"

**Comment by YangZxi** [comment]
previous name is yangzx1282

**Comment by DSDmark** [comment]
Hi @yangkokoi,

Go to settings and click 0n **Account** section, then there is a change username section. You can change your username.

Here is quick link:-https://github.com/settings/admin

**Comment by radar-blip** [comment]
There's a chance Github will let you reclaim your old username since it was yours previously and not someone elses.
Explain the situation and ask for help, see link below:

For account related issues, see here: [Support page](https://support.github.com/contact/account)

**Comment by 1rom11** [comment]
Maybe created a respiratory called yangkokoi/yangkokoi then change your username via [Admin panel](https://github.com/settings/admin)
